[2025-04-28T00-43-15-211Z] ERROR:
Name: Error
Message: UNHANDLED_REJECTION
Stack:
Error: UNHANDLED_REJECTION
    at Object.error (/home/container/events/logger.js:152:27)
    at process.<anonymous> (/home/container/index.js:210:10)
    at process.emit (node:events:518:28)
    at emitUnhandledRejection (node:internal/process/promises:252:13)
    at throwUnhandledRejectionsMode (node:internal/process/promises:388:19)
    at processPromiseRejections (node:internal/process/promises:475:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:106:32)
Metadata: {
  "requestBody": {
    "files": [],
    "json": {
      "type": 4,
      "data": {
        "content": "Error al procesar la interacci√≥n",
        "tts": false,
        "enforce_nonce": false,
        "flags": 64
      }
    }
  },
  "rawError": {
    "message": "Unknown interaction",
    "code": 10062
  },
  "code": 10062,
  "status": 404,
  "method": "POST",
  "url": "https://discord.com/api/v10/interactions/1366213137216442498/aW50ZXJhY3Rpb246MTM2NjIxMzEzNzIxNjQ0MjQ5ODpQd2ExWmFrTkRoZHVkbVlRSVhNU1ptcXB6MDlIWGNpRUJWOU1TaURiTHY2U3RGdU80SnV5MmpSaHlQbVowdVJMempMc0Y4S2lvdjd2a0lobDYzNHV3NzU3V0hEbjl6blVQeVdkbTRTbFVoVDlLY21pS2dxSHBPdlAzenFMNXFjbQ/callback?with_response=false"
}

